From: Johannes Schauer Marin Rodrigues <josch@debian.org>
Subject: avoid redefinition of MIN and MAX

--- a/CORE/SERVICES/COMMON/osdep_adf.h
+++ b/CORE/SERVICES/COMMON/osdep_adf.h
@@ -962,8 +962,12 @@ typedef struct scatterlist            sg
 #define ARRAY_LENGTH(a)         (sizeof(a) / sizeof((a)[0]))
 #endif
 
+#ifndef MIN
 #define MIN(a, b)                adf_os_min(a, b)
+#endif
+#ifndef MAX
 #define MAX(a, b)                adf_os_max(a, b)
+#endif
 
 /*
  * PCI configuration space access
